lib/basquiat/version.rb

```diff
@@ -2,5 +2,5 @@
 
 # Version file
 module Basquiat
-  VERSION = '1.3.6'
+  VERSION = '1.3.7'
```

Comment:
I believe that you can increase major version to 4, because it`s not a bug fix, but you are including a small feature.

Reply:
Done :)